File: Basic/CPP/problem_4.cpp
// C++ Program to Calculate Area of Square

#include <iostream>
using namespace std;

int main(){
    int side, area;
    
    // Asking for input
    cout << "Enter the side of square: ";
    cin >> side;
    
    // Calculating area
    area = side * side;
    
    // Displaying output
    cout << "Area of square of side " << side << " is: " << area;
    return 0;
}

File: Basic/Java/Problem_4.java
//Write a program to input the side and print the area of square. Area of square = side*side.

import java.util.Scanner;

public class AreaS
{
   public static void main(String[] args)
   {
      float side, area;
      Scanner s = new Scanner(System.in);
      
      System.out.print("Enter the Side Length of Square: ");
      side = s.nextFloat();
      
      area = side*side;
      System.out.println("\nArea = " +area);
   }
}
